src/ansys/sherlock/core/common.py

```diff
@@ -113,3 +113,20 @@ def list_units(self, unitType):
             raise e
 
         return response.units
+
+    def get_solders(self):
+        """List valid solders.
```

Comment:
Change the wording to be consistent with other get methods.
E.g.:
https://sherlock.docs.pyansys.com/version/stable/api/_autosummary/ansys.sherlock.core.stackup.Stackup.get_layer_count.html